packages/openapi-v3/src/controller-spec.ts

```diff
@@ -239,3 +285,7 @@ export function getControllerSpec(constructor: Function): ControllerSpec {
   }
   return spec;
 }
+
+function getModelRef(fn: Function) {
+  return {$ref: `#/components/schemas/${fn.name}`};
```

Comment:
If `fn` is a Model class with `@model({name: 'my-model'})`, we should use `my-model`.

Reply:
It would be great if we could share the implementation of "Model class to $ref" algorithm with openapi-v3 package, see https://github.com/strongloop/loopback-next/blob/8499862ed7bbb615d8bcdf2c2590fcdfaa624e8f/packages/openapi-v3/src/generate-schema.ts#L54-L65